GetARock/GetARock/Screens/BandCreation/BandMemberAddFlow/UserSearch+Invite/AddedBandMemberCell.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  AddedBandMemberCell.swift
+//  GetARock
+//
+//  Created by Jisu Jang on 2023/02/16.
+//
+
+import UIKit
+
+final class AddedBandMemberCell: UICollectionViewCell, Identifiable {
+
+    // MARK: - properties
+    var id: String = "default"
+
+    private let backgroundContentView: UIView = {
+        let view = UIView()
+        view.layer.masksToBounds = true
+        view.backgroundColor = .activeGradationPurple
```

Comment:
p2;
```suggestion
        view.applyMainGradient()
```
로 바꿔주시면 UI가 깨지지 않습니다!

Reply:
헉 감사합니다! 반영완료!!  https://github.com/extreme-rock/GetARock-iOS/pull/128/commits/ab5dd9f50bc8f7206ab53d6181c809d609ca2460